Dockerfile

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2018 Expedia Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+
+FROM amazonlinux:latest
+
+# Versions
+ARG HADOOP_VERSION=2.7.2
+ARG HIVE_VERSION=2.3.4
+ENV HADOOP_VERSION=${HADOOP_VERSION}
+ENV HIVE_VERSION=${HIVE_VERSION}
+
+ENV JAVA_VERSION 1.8.0
+ENV SHUNTING_YARD_VERSION 0.0.6
+ENV CIRCUS_TRAIN_VERSION 13.2.1
+ENV SHUNTING_YARD_HOME /opt/shunting-yard
+ENV CIRCUS_TRAIN_HOME /opt/circus-train
+
+COPY files/RPM-GPG-KEY-emr /etc/pki/rpm-gpg/RPM-GPG-KEY-emr
+COPY files/emr-apps.repo /etc/yum.repos.d/emr-apps.repo
+COPY files/emr-platform.repo /etc/yum.repos.d/emr-platform.repo
+
+RUN yum -y update && \
+  yum install -y java-${JAVA_VERSION}-openjdk \
+    procps \
+    awscli \
+    util-linux \
+    tar \
+    which \
+    jq \
+    wget \
+    hadoop \
+    hive \
+  && yum clean all \
+  && rm -rf /var/cache/yum
+
+ENV JAVA_HOME /usr/lib/jvm/jre-1.8.0-openjdk/
+ENV PATH $PATH:$JAVA_HOME/bin
+
+ENV HADOOP_HOME /usr/lib/hadoop
+ENV PATH $PATH:$HADOOP_HOME/bin
+
+ENV HIVE_HOME /usr/lib/hive
+ENV PATH $PATH:$HIVE_HOME/bin
+
+ENV HIVE_LIB /usr/lib/hive/lib/
```

Comment:
Where are you using this Env?

Reply:
All these ENV Variables are used here: https://github.com/HotelsDotCom/shunting-yard/blob/master/shunting-yard-binary/src/main/scripts/replicator.sh